tests/Api/TestCase.php

```diff
@@ -0,0 +1,28 @@
+<?php
+
+namespace Mailgun\Tests\Api;
+
+/**
+ * @author Tobias Nyholm <tobias.nyholm@gmail.com>
+ * @author Contributors of https://github.com/KnpLabs/php-github-api
+ */
+abstract class TestCase extends \PHPUnit_Framework_TestCase
+{
+    abstract protected function getApiClass();
+
+    protected function getApiMock()
+    {
+        $httpClient = $this->getMockBuilder('Http\Client\HttpClient')
+            ->setMethods(array('sendRequest'))
+            ->getMock();
+        $httpClient
+            ->expects($this->any())
+            ->method('sendRequest');
+
+        $client = new \Mailgun\Mailgun('api-key', $httpClient);
+        return $this->getMockBuilder($this->getApiClass())
+            ->setMethods(array('get', 'post', 'postRaw', 'delete', 'put'))
+            ->setConstructorArgs(array($client))
+            ->getMock();
+    }
+}
```

Comment:
Minor update - Code Styling:
Can you add here a new empty line at the EOF please?